? twfy/www/docs/images/mps
? twfy/www/docs/images/mpsL
Index: twfy/www/docs/index-election.php
===================================================================
RCS file: /repos/mysociety/twfy/www/docs/index-election.php,v
retrieving revision 1.2
diff -u -r1.2 index-election.php
--- twfy/www/docs/index-election.php	10 Aug 2007 11:09:15 -0000	1.2
+++ twfy/www/docs/index-election.php	29 Nov 2007 02:59:31 -0000
@@ -102,7 +102,7 @@
 	?>
 						<form action="<?php echo $MPURL->generate(); ?>" method="get">
     <p><strong>Now that the election has been called, you can use this site to find out what your ex-MP did throughout the last parliament. We have performance stats, speeches, voting records and more...</strong><br>
-						<label for="pc">Enter your UK postcode here:</label>&nbsp; <input type="text" name="pc" id="pc" size="8" maxlength="10" value="<?php echo htmlentities($THEUSER->postcode()); ?>" class="text">&nbsp;&nbsp;<input type="submit" value=" GO " class="submit"></p>
+						<label for="pc">Enter your Australian postcode here:</label>&nbsp; <input type="text" name="pc" id="pc" size="8" maxlength="10" value="<?php echo htmlentities($THEUSER->postcode()); ?>" class="text">&nbsp;&nbsp;<input type="submit" value=" GO " class="submit"></p>
 						</form>
 <?php
 }
Index: twfy/www/docs/index.php
===================================================================
RCS file: /repos/mysociety/twfy/www/docs/index.php,v
retrieving revision 1.11
diff -u -r1.11 index.php
--- twfy/www/docs/index.php	10 Aug 2007 11:09:15 -0000	1.11
+++ twfy/www/docs/index.php	29 Nov 2007 02:59:31 -0000
@@ -59,12 +59,12 @@
 
 	if ($pc_form) { ?>
 		<form action="<?php echo $MPURL->generate(); ?>" method="get">
-		<p><strong>Find out more about your MP</strong><br>
-		<label for="pc">Enter your UK postcode here:</label>&nbsp; <input type="text" name="pc" id="pc" size="8" maxlength="10" value="<?php echo htmlentities($THEUSER->postcode()); ?>" class="text">&nbsp;&nbsp;<input type="submit" value=" GO " class="submit"></p>
+		<p><strong>Find out more about your Representative</strong><br>
+		<label for="pc">Enter your Australian postcode here:</label>&nbsp; <input type="text" name="pc" id="pc" size="8" maxlength="10" value="<?php echo htmlentities($THEUSER->postcode()); ?>" class="text">&nbsp;&nbsp;<input type="submit" value=" GO " class="submit"></p>
 		</form>
 	<?php
 		if (!defined("POSTCODE_SEARCH_DOMAIN")) {
-			print '<p align="right"><em>Postcodes are being mapped to a random MP</em></p>';
+			print '<p align="right"><em>Postcodes are being mapped to a random Representative</em></p>';
 		}
 	}
 	print "</li>";
Index: twfy/www/docs/api/api_getMP.php
===================================================================
RCS file: /repos/mysociety/twfy/www/docs/api/api_getMP.php,v
retrieving revision 1.10
diff -u -r1.10 api_getMP.php
--- twfy/www/docs/api/api_getMP.php	19 Oct 2007 09:24:10 -0000	1.10
+++ twfy/www/docs/api/api_getMP.php	29 Nov 2007 02:59:31 -0000
@@ -36,13 +36,13 @@
 		$row['last_name'], $row['constituency']);
 	if (isset($parties[$row['party']]))
 		$row['party'] = $parties[$row['party']];
-	if (is_file(BASEDIR . IMAGEPATH . 'mpsL/' . $row['person_id'] . '.jpg')) {
+	if (is_file(FILEIMAGEPATH . 'mpsL/' . $row['person_id'] . '.jpg')) {
 		$row['image'] = IMAGEPATH . 'mpsL/' . $row['person_id'] . '.jpg';
-	} elseif (is_file(BASEDIR . IMAGEPATH . 'mpsL/' . $row['person_id'] . '.jpeg')) {
+	} elseif (is_file(FILEIMAGEPATH . 'mpsL/' . $row['person_id'] . '.jpeg')) {
 		$row['image'] = IMAGEPATH . 'mpsL/' . $row['person_id'] . '.jpeg';
-	} elseif (is_file(BASEDIR . IMAGEPATH . 'mps/' . $row['person_id'] . '.jpg')) {
+	} elseif (is_file(FILEIMAGEPATH . 'mps/' . $row['person_id'] . '.jpg')) {
 		$row['image'] = IMAGEPATH . 'mps/' . $row['person_id'] . '.jpg';
-	} elseif (is_file(BASEDIR . IMAGEPATH . 'mps/' . $row['person_id'] . '.jpeg')) {
+	} elseif (is_file(FILEIMAGEPATH . 'mps/' . $row['person_id'] . '.jpeg')) {
 		$row['image'] = IMAGEPATH . 'mps/' . $row['person_id'] . '.jpeg';
 	}
 	$row = array_map('html_entity_decode', $row);
Index: twfy/www/docs/hansard/index.php
===================================================================
RCS file: /repos/mysociety/twfy/www/docs/hansard/index.php,v
retrieving revision 1.4
diff -u -r1.4 index.php
--- twfy/www/docs/hansard/index.php	16 May 2007 15:26:34 -0000	1.4
+++ twfy/www/docs/hansard/index.php	29 Nov 2007 02:59:31 -0000
@@ -82,7 +82,7 @@
 $PAGE->stripe_end();
 $PAGE->stripe_start();
 ?>
-				<h3>Busiest House of Commons debates from the most recent week</h3>
+				<h3>Busiest House of Representatives debates from the most recent week</h3>
 <?php
 $DEBATELIST = new DEBATELIST;
 $DEBATELIST->display('biggest_debates', array('days'=>7, 'num'=>$number_of_debates_to_show));
@@ -104,124 +104,5 @@
 	)
 ));
 
-$PAGE->stripe_start();
-?>
-				<h3>Busiest House of Lords debates from the most recent week</h3>
-<?php
-$DEBATELIST = new LORDSDEBATELIST;
-$DEBATELIST->display('biggest_debates', array('days'=>7, 'num'=>$number_of_debates_to_show));
-
-$MOREURL = new URL('lordsdebatesfront');
-$anchor = $number_of_debates_to_show + 1;
-?>
-				<p><strong><a href="<?php echo $MOREURL->generate(); ?>#d<?php echo $anchor; ?>">See more debates</a></strong></p>
-<?php
-
-$PAGE->stripe_end(array(
-	array (
-		'type' => 'include',
-		'content' => "holdebates_short"
-	),
-	array (
-		'type' => 'include',
-		'content' => "calendar_holdebates"
-	)
-));
-
-$PAGE->stripe_start();
-?>
-				<h3>Some recent written answers</h3>
-<?php
-
-$WRANSLIST = new WRANSLIST;
-$WRANSLIST->display('recent_wrans', array('days'=>7, 'num'=>$number_of_wrans_to_show));
-
-$MOREURL = new URL('wransfront');
-?>
-				<p><strong><a href="<?php echo $MOREURL->generate(); ?>">See more written answers</a></strong></p>
-<?php
-
-$PAGE->stripe_end(array(
-	array (
-		'type' => 'include',
-		'content' => "wrans_short"
-	),
-	array (
-		'type' => 'include',
-		'content' => "calendar_wrans"
-	)
-));
-
-$PAGE->stripe_start();
-
-?>
-				<h3>Busiest Westminster Hall debates from the most recent week</h3>
-<?php
-$WHALLLIST = new WHALLLIST;
-$WHALLLIST->display('biggest_debates', array('days'=>7, 'num'=>$number_of_debates_to_show));
-
-$MOREURL = new URL('whallfront');
-$anchor = $number_of_debates_to_show + 1;
-?>
-				<p><strong><a href="<?php echo $MOREURL->generate(); ?>#d<?php echo $anchor; ?>">See more debates</a></strong></p>
-<?php
-
-$PAGE->stripe_end(array(
-	array (
-		'type' => 'include',
-		'content' => "whalldebates_short"
-	),
-	array (
-		'type' => 'include',
-		'content' => "calendar_whalldebates"
-	)
-));
-
-$PAGE->stripe_start();
-
-?>
-				<h3>Some recent Written Ministerial Statements</h3>
-<?php
-$WMSLIST = new WMSLIST;
-$WMSLIST->display('recent_wms', array('days'=>7, 'num'=>$number_of_wrans_to_show));
-$MOREURL = new URL('wmsfront');
-?>
-				<p><strong><a href="<?php echo $MOREURL->generate(); ?>">See more written ministerial statements</a></strong></p>
-<?php
-$PAGE->stripe_end(array(
-	array( 'type' => 'include', 'content' => 'wms_short' ),
-	array( 'type' => 'include', 'content' => 'calendar_wms' )
-));
-
-$PAGE->stripe_start();
-?>
-	<h3>Busiest Northern Ireland Assembly debates from the most recent month</h3>
-<?php
-$DEBATELIST = new NILIST;
-$DEBATELIST->display('biggest_debates', array('days'=>30, 'num'=>$number_of_debates_to_show));
-
-$MOREURL = new URL('nidebatesfront');
-$anchor = $number_of_debates_to_show + 1;
-?>
-		<p><strong><a href="<?php echo $MOREURL->generate(); ?>#d<?php echo $anchor; ?>">See more debates</a></strong></p>
-<?php
-
-$PAGE->stripe_end(array(
-	array (
-		'type' => 'include',
-		'content' => "nidebates_short"
-	),
-	array (
-		'type' => 'include',
-		'content' => "calendar_nidebates"
-	)
-));
-$PAGE->stripe_start();
-?>
-
-<p>Still to come: Select Committees, and much more...</p>
-
-<?php
-$PAGE->stripe_end();
 $PAGE->page_end();
 ?>
Index: twfy/www/docs/mp/.htaccess
===================================================================
RCS file: /repos/mysociety/twfy/www/docs/mp/.htaccess,v
retrieving revision 1.5
diff -u -r1.5 .htaccess
--- twfy/www/docs/mp/.htaccess	11 Dec 2006 00:37:26 -0000	1.5
+++ twfy/www/docs/mp/.htaccess	29 Nov 2007 02:59:31 -0000
@@ -1,5 +1,4 @@
 RewriteEngine on
-RewriteBase /mp/
 RewriteRule ^c4/(['a-zA-Z_+-]+)/([\(\)\.a-zA-Z_+,'-]+)$ index.php?c4=1&n=$1&c=$2
 RewriteRule ^c4x/(['a-zA-Z_+-]+)/([\(\)\.a-zA-Z_+,'-]+)$ index.php?c4x=1&n=$1&c=$2
 RewriteRule ^(['a-zA-Z_+-]+)/([\(\)\.a-zA-Z_+,'-]+)$ index.php?n=$1&c=$2 [QSA]
Index: twfy/www/docs/news/.htaccess
===================================================================
RCS file: /repos/mysociety/twfy/www/docs/news/.htaccess,v
retrieving revision 1.3
diff -u -r1.3 .htaccess
--- twfy/www/docs/news/.htaccess	11 Dec 2006 00:37:26 -0000	1.3
+++ twfy/www/docs/news/.htaccess	29 Nov 2007 02:59:31 -0000
@@ -1,5 +1,4 @@
 RewriteEngine on
-RewriteBase /news/
 #RewriteRule ^mps/c4/$ mps/?c4=1 [QSA]
 RewriteRule ^archives/(.*)$ index.php/archives/$1 [QSA]
 RewriteRule ^index.rdf$ rdf.php [QSA]
Index: twfy/www/docs/news/editme.php
===================================================================
RCS file: /repos/mysociety/twfy/www/docs/news/editme.php,v
retrieving revision 1.8
diff -u -r1.8 editme.php
--- twfy/www/docs/news/editme.php	19 Oct 2007 09:23:43 -0000	1.8
+++ twfy/www/docs/news/editme.php	29 Nov 2007 02:59:31 -0000
@@ -4,436 +4,17 @@
 
 $all_news = array(
 
-43 => array('The Queen in Parliament', <<<EOT
-Whilst we cover MPs and Lords, there's always been one other individual who speaks in
-Parliament whom we missed. That anomaly has now been fixed, and so you can now view
-<a href="http://www.theyworkforyou.com/royal/elizabeth_the_second">the Queen's
-page</a> on TheyWorkForYou. :)
-This means you can sign up for an email alert whenever she speaks
-in Parliament, subscribe to the
-<a href="http://www.theyworkforyou.com/rss/mp/13935.rdf">RSS feed</a> of the same,
-or just look at past
-<a href="http://www.theyworkforyou.com/search/?pid=13935&pop=1">prorogation and Queen's speeches</a>
-more easily than anywhere else I've found.
+1 => array('Birth of an Australian TheyWorkForYou', <<<EOT
+By hacking, levering, doing terrible things while nobody is looking,
+pummeling and other things in that general direction, we've managed
+to somehow get a very crude version of an Australian version of
+TheyWorkForYou up and running. All credit goes to the UK guys for
+putting so much love and labour into a codebase that made this all
+really surprisingly straightforward. Thank you!
 EOT
-, '2007-09-11 15:34:47', 'Matthew'),
+, '2007-11-22 17:46:00', 'Matthew'),
 
-42 => array('Missing confirmation emails', <<<EOT
-Whilst doing some routine maintenance, I spotted that due to a
-server configuration mistake, TheyWorkForYou has been
-failing to send out any confirmation emails since mid-August.
-Very many apologies to anyone affected - emails from the past
-few days have now been sent out, and I've got in touch
-with anyone who signed up in the days prior to
-that to let them know what happened.
-EOT
-, "2007-09-03 17:12:47", 'Matthew'),
-
-41 => array('The Northern Ireland Assembly', <<<EOT
-I'm extremely proud to announce that TheyWorkForYou now covers
-the debates of the Northern Ireland Assembly from its
-<a href="/ni/?d=1998-07-01">creation in 1998</a> to its current
-<a href="/ni/?d=2006-12-04">Transitional Assembly</a> status.
-Everything should integrate with the rest of the site, so
-MLAs who are also MPs only have the one central page, such as
-<a href="/mp/gerry_adams/belfast_west">Gerry Adams</a> or
-<a href="/mp/ian_paisley/north_antrim">Ian Paisley</a>.
-
-I've done this voluntarily, in secret &mdash;
-this announcement comes as much as a surprise to everyone else involved with
-TheyWorkForYou as it does to you &mdash;
-partly to try out 
-<a title="The moment I realised I should simply make &lt;b&gt;, &lt;i&gt;, and &lt;font&gt; nestable to get everything to parse was quite good"
- href="http://www.crummy.com/software/BeautifulSoup/">Beautiful Soup</a>,
-partly because I obviously didn't have enough to do writing a
-<a href="http://petitions.pm.gov.uk/">No10 Petitions</a> website and
-answering mountains of support mail, partly as an early Christmas present for everyone I know in the world
-of the internets (hey guys), and partly to show how easy I think it is and that there's no excuse
-for people not to volunteer to do the Scottish Parliament
-and Welsh Assembly. :) 
-<a href="http://project.knowledgeforge.net/ukparse/trac/browser/trunk/parlparse/pyscraper/ni">Here's all the code</a>
-(and there's not much; the scraper is 28 lines long, the name resolver 273, and the parser only 343 &mdash; including comments!)
-for scraping and parsing the Northern Ireland Assembly,
-<a href="http://project.knowledgeforge.net/ukparse/trac/browser/trunk/parlparse/members">the XML list of Assembly Members</a>
-I created was quite important, and then it was a matter of adapting the existing
-<a href="https://secure.mysociety.org/cvstrac/dir?d=mysociety/twfy">TheyWorkForYou code</a> to add
-a new major type (and completely rewrite the enter/leaving code as that assumed you couldn't be
-in more than one Parliament/Assembly at once...).
-
-I'm sure there are bugs around the place (<a href="/contact/">do let us know</a>), but I'm off on holiday for a week now;
-I look forward to my return (so don't expect a quick reply to suggestions or
-complaints)! :-)
-
-ATB,<br>
-<a href="http://www.dracos.co.uk/">Matthew Somerville</a>
-EOT
-, "2006-12-11 00:00:00", 'Matthew'),
-
-40 => array('TheyWorkForYou API', <<<EOT
-Today, we launched an API (Application Programming Interface) to the
-data TheyWorkForYou contains. This lets other sites query our site for
-various bits of information, such as the constituency or MP for a particular
-postcode, the debates for a particular date, the comments left on a certain
-MP's speeches, and so on. It's not complete
-and will hopefully have improvements in future &ndash;
-<a href="/api/">see the
-documentation and have a play with it</a>. Do dive in, and <a href="/contact/">let us 
-know</a> of any interesting or fun applications you come up with! The
-immediate new feature on TheyWorkForYou itself is the list of
-neighbouring constituencies on an MP's page.
-
-Here's my <a href="http://www.mysociety.org/2006/barcamp-london/">BarCamp
-Presentation</a> launching the API, and a <a href="http://www.dracos.co.uk/work/theyworkforyou/api/fabfarts/">couple</a>
-of <a href="http://www.mysociety.org/2006/09/02/battle-your-way-to-sedgefield/">applications</a> using it.
-EOT
-, "2006-09-02 17:10:00"),
-
-39 => array('New features over the last couple of months', <<<EOT
-Just a brief round-up of what we've added since our last piece of news.
-
-We've added MPs' membership of select committees, taken from
-<a href="http://www.parliament.uk/directories/hciolists/selmem.cfm">this
-official list</a>, including membership history where we can work it out.
-
-Our rankings for various numbers have been removed, to be replaced by
-woolly text &ndash; we have <a href="http://www.theyworkforyou.com/help/#numbers">more
-information on this</a>. To make up for it, we've done a bit more text
-analysis, and added which departments and subjects an MP asks most
-written questions about, so you can see that
-<a href="http://www.theyworkforyou.com/mp/alan_williams/swansea_west">Alan Williams</a>
-has asked more questions about Royal Residences than anything else, or that
-<a href="http://www.theyworkforyou.com/mp/anne_main/st_albans">Anne Main</a> has asked
-more questions on Flag Flying.
-
-Firefox users can download a
-<a href="http://mycroft.mozdev.org/download.html?name=theyworkforyou">search plugin</a>,
-to enable easy searching of the site from Firefox's search bar.
-
-And a few bugfixes - our handling of when Oral Answers finish should be a bit better than previously, 
-and when searching a particular person's speeches, we no longer group the results by debate, which
-was confusing, as you obviously expected to see all the speeches for that person that matched your
-search terms.
-EOT
-, "2006-08-18 09:46:36"),
-
-38 => array("Got an idea for a useful website?", <<<EOT
-mySociety is running a call for proposals until 16th June. If you can
-come up with an idea for a useful site like this one, and it beats all
-the other ideas, we'll build it for free. <a href="http://www.mysociety.org/proposals2006/submit">Submit an idea</a>
-or <a href="http://www.mysociety.org/proposals2006/view">read and comment on what other people have submitted</a>.
-EOT
-, "2006-06-06 06:06:06"),
-
-37 => array("We've added the Lords, and more", <<<EOT
-For your use and enjoyment we've added Hansard for the House of Lords (their debates,
-written questions and ministerial statements, just like the Commons), and a page
-on each of the <a href="/peers/">members of the House of Lords</a>. That isn't all - we've
-also added a feature to the search to help you identify which MPs or
-Lords are interested in a certain word or phrases. Just search as
-usual and then click 'Show use by person'.
-EOT
-, "2006-06-01 12:34:56"),
-
-36 => array("Changes to the Register of Members' Interests", <<<EOT
-One of the many things our site does, probably without you realising, is to track the Register of Members' Interests, in which MPs are "to provide information of any pecuniary interest or other material benefit which a Member receives which might reasonably be thought by others to influence his or her actions, speeches or votes in Parliament, or actions taken in the capacity of a Member of Parliament". The latest entry published on the official site has always been shown on MP pages (<a href="http://www.theyworkforyou.com/mp/tony_blair/sedgefield#register">here's Tony Blair's</a>), however we kept all previous editions of the Register safe and sound. And now, after a bit of coding, you can view a history of the Register, either <a href="http://www.theyworkforyou.com/regmem/?f=2005-12-14">comparing particular editions</a>, or for particular MPs (<a href="http://www.theyworkforyou.com/regmem/?p=10001">Diane Abbott</a>, for example). Entries only have to stay on the Register for a year, so this can make for some interesting reading.
-
-Happy New Year! :) <a href="http://www.theyworkforyou.com/regmem/">Changes to the Register of Members' Interests</a> 
-EOT
-, "2006-01-01 12:36:00"),
-
-35 => array("TheyWorkForYou.com Wins Award", <<<EOT
-We're proud to announce that TheyWorkForYou.com has won the  'Contribution to Civic Society' award at the <a href="http://www.newstatesman.co.uk/nma/nma2005/nma2005index.php">New Statesman magazine's New Media Awards</a>.
-
-Thanks to the kind souls who nominated us. We enjoyed ourselves.
-
- 
-EOT
-, "2005-07-05 23:54:13"),
-
-34 => array("NEW! Email this page to a friend", <<<EOT
-At the top right hand side of every (ex)MP's page, we've added a simple new feature: email the page to a friend.
-
-<a href="/mp/">Have a go</a>, and let all your friends know how easy it is to keep tabs on what your former MP did and said in your name during the last Parliament. 
-EOT
-, "2005-04-14 23:43:50"),
-
-33 => array("Play with Pledgebank, Pound the Streets", <<<EOT
-Our good friends over at <a href="http://www.mysociety.org/">MySociety.org</a> are close to launching <a href="http://www.pledgebank.com/">Pledgebank</a>, a new web and sms service which  does what it says on the url.
-
-To help test the concept,  they've set up a <a href="http://www.pledgebank.com/theywork">special TheyWorkForYou.com pledge</a> which reads:
-<blockquote>
-<i>"I will deliver a printed copy of my local MP's page on TheyWorkForYou.com to every house in my street but only if 100 other fans of TheyWorkForYou.com will too."
-</i>
-</blockquote>
-
-If you're up for it, why not <a href="http://www.pledgebank.com/theywork">sign up now?</a>  
-EOT
-, "2005-04-13 22:41:23"),
-
-32 => array("NEW! How MPs voted on Key Issues", <<<EOT
-We've added a smashing new feature to all our <a href="/mps/">MP pages</a>, courtesy of Francis at <a href="http://www.publicwhip.org.uk">PublicWhip</a>
-
-You can now check out which way MPs tended vote on half a dozen key issues from the last Parliament (Iraq, Fox Hunting, ID cards. Top Up Fees etc) via a nice, easy to understand panel. 
-
-For example, mining the <a href="http://www.theyworkforyou.com/mp/graham_allen/nottingham_north#votingrecord">voting record of Graham Allen</a>, Labour MP for Nottingham North, shows that he was pretty dubious about the Iraq war, but very strongly in favour of a ban on fox hunting.
-
-This information is the result sophisticated mining of PublicWhip's detailed vote data covering hundreds of Parliamentary divisions.
-
-We hope it'll make it easier for you to keep track of where your MP really stands on key issues - after all, They Work For You. 
-EOT
-, "2005-04-05 22:59:57"),
-
-31 => array("Channel 4 linking to TheyWorkForYou.com", <<<EOT
-We're happy to announce that the Channel 4 website is directing people to TheyWorkForYou.com's MP pages via a <a href="http://www.channel4.com/news/microsites/E/election2005/yourmp.html">postcode box</a> on their <a href="http://www.channel4.com/news/microsites/E/election2005/">Election 2005 website</a>.
-
-We've done a special paint job for users coming from the Channel 4 site, just to make them feel welcome. You can see an example <a href="http://www.theyworkforyou.com/mp/c4/paul_keetch/hereford">here</a>.
-
-We'd love to know what you think of this paint job, as we're planning a redesign. As ever, <a href="/contact/">get in touch</a>.
-EOT
-, "2005-04-05 22:53:32"),
-
-30 => array("NEW! Email Alerts & Other Features", <<<EOT
- Since launching in June, we've made hundreds of small
-     improvements to the site. However, the cold winter nights
-     have sparked an explosion of activity, and we've launched a
-     slew of new features in response to requests such as:
-
-     <i>"Can you Email me when my MP next speaks, or when an issue
-       I care about is raised?"</i>
-
-     You all *so* wanted this. Near the top of every MP's page,
-     and on every search results page, you'll see a link starting
-     with 'Email me when...' Just click and go, or sign up by
-     hand using <a href="http://theyworkforyou.com/alert/">http://theyworkforyou.com/alert/</a>
-
-    <i> "Can I see when an issue was last raised in Parliament?"</i>
-
-     Yes. At the top of every search results page you'll see a
-     link that sorts the most recent result at the top. Ideal for
-     keeping tabs on topical concerns such as
-     <a href="http://theyworkforyou.com/search/?o=d&s=%22identity+card%22">http://theyworkforyou.com/search/?o=d&s=%22identity+card%22</a>
-
-     <i>"Can I just search the stuff my MP has said?"</i>
-
-     Yup. Go to an MP's page. See the red search box to the right
-     of your MP's delicious photo? That's your baby. It'll search
-     just that MP's contributions.
-
-     <i>"What about Westminster Hall, Written Ministerial
-       Statements & House of Commons' Committees?"</i>
-
-     We're two-thirds done. See <a href="http://theyworkforyou.com/whall/">http://theyworkforyou.com/whall/</a> and
-     <a href="http://theyworkforyou.com/wms/">http://theyworkforyou.com/wms/</a> We're busy tacking Committee
-     proceedings, but these be hard.
-
-     <i>"Is what you're doing legal?"</i>
-
-     Yes. We are legit. Indeed, within a couple of months of launch
-     we had Parliament's blessing, plenty of enthusiasm and a nice
-     shiny licence to re-use Hansard. Given Parliament's history &
-     traditions, such a swift & positive reaction is most admirable.
-
-     FINALLY, TELL YOUR FRIENDS
-
-     TheyWorkForYou.com is the best place to get the unadulterated
-     lowdown on what your MP has said and done in your name.
-
-     As the election approaches, we think the site could make a
-     real difference to democratic transparency and engagement.
-
-     If you agree, please tell your friends. Blog about us. Write
-     about us. Link to us. Use our RSS feeds in your sites. Tell
-     your enemies. Hell, even tell your parents. You're the only
-     marketing we can afford!
-
-     Best wishes,
-
-    - Tom, on behalf of the TheyWorkForYou.com volunteers
-
-     <a href="http://theyworkforyou.com/about/">http://theyworkforyou.com/about/</a>  - New volunteers welcome! 
-EOT
-, "2005-02-24 22:02:44"),
-
-26 => array("New Release of TheyWorkForYou.com Source Code", <<<EOT
-<strong>Update:</strong> New URL for the code.
-
-We've released version 8 of the <a href="https://secure.mysociety.org/cvstrac/dir?d=mysociety/twfy">TheyWorkForYou.com source code</a> which is available for download under an Open Source licence.
-
-The update contains new code for:
-
-- Westminster Hall Debates
-- Written ministerial statements
-- Better search code
-
-Feel free to download it and have a play.
-
-Whilst you're at it, don't forget we also publish a full <a href="http://www.theyworkforyou.com/raw/">XML version of the Hansard data</a>. 
-EOT
-, "2005-01-25 00:15:19"),
-
-25 => array("Search by Date", <<<EOT
-Thanks to the sterling efforts of a fine new volunteer (thanks David!), you can now sort any search results either by listing the <a href="http://www.theyworkforyou.com/search/?s=Uzbekistan&o=r">most relevant results first</a>, or <a href="http://www.theyworkforyou.com/search/?s=Uzbekistan&o=d">most recent results first</a>.
-
-
-This is good news for those wanting to keep up to date with issues as they pop up in the Commons.
 
-Next step on search is probably an RSS version of keyword searches. We're still very keen to improve overall relevancy of results - anyone out there fancy lending us a Google Appliance to play with? 
-
-As ever, <a href="/contact/">contact us</a> if you've got any suggestions how we might improve the site, or if you fancy volunteering. 
-EOT
-, "2005-01-25 00:16:09"),
-
-20 => array("New! Ministerial Statements now included", <<<EOT
-The <a href="http://www.theyworkforyou.com/">TheyWorkForYou.com</a> volunteers are busily crunching their way through the more esoteric corners of Hansard. By way of example, we've just incorporated <a href="http://www.theyworkforyou.com/wms/">Written Ministerial Statements</a> into the site.
-
-Nope, most of us had never heard of them either, but apparently:
-
-<i><a href="http://www.theyworkforyou.com/wms/">Written Ministerial Statements</a> were introducted in late 2002 to stop the practice of having "planted" or "inspired" questions designed to elicit Government statements.
-
-They are just that - statements on a particular topic by a Government Minister.
-</i>
-
-Well, there you go. You live and learn. 
-
-Next up: Select Committees, which promise to be an order of magnitude more challenging. We'll keep you posted.
-
-PS We're always on the lookout for new volunteers - <a href="/contact/">contact us</a> if you are keen to help in almost any capacity. 
-EOT
-, "2005-01-17 17:44:47"),
-
-19 => array("NEW! Westminster Hall debates now available.", <<<EOT
-The volunteers have been busy over the past few days, and we are now one stage closer towards <a href="http://www.theyworkforyou.com/">TheyWorkForYou.com</a> becoming a truly comprehensive record of Parliamentary activity.
-
-We've included <a href="http://www.theyworkforyou.com/whall/">debates from Westminster Hall</a>, which is a new-ish forum sitting in parallel to the main Commons Chamber. Learn more about it <a href="http://www.explore.parliament.uk/Parliament.aspx?id=10416&glossary=true">here</a>.
-
-<a href="http://www.theyworkforyou.com/whall/">Westminster Hall Debates</a> were introduced in 1999 with the aim of encouraging constructive rather than confrontational debate between MPs. 
-
-As ever, we'll let you be the judge. 
-
-Next up, parsing all the various House of Commons' Committees... .
-
-
-
-
- 
-EOT
-, "2004-12-22 23:47:19"),
-
-18 => array("RSS feed of your MP's recent appearances", <<<EOT
-Don't forget you can keep track of any MP's recent House of Commons appearances via an RSS newsfeed. You'll find the link in the right-hand column of each individual MP's page.
-
-For example, the RSS feed of <a href="http://www.theyworkforyou.com/mp/?pid=10508">Barabara Roche's</a> recent appearances in the House of Commons can be found at the following web address:
-
-<blockquote><a href="http://www.theyworkforyou.com/rss/mp/10508.rdf">http://www.theyworkforyou.com/rss/mp/10508.rdf</a>
-</blockquote>
-
-RSS is a simple way to publish & distribute content which is frequently updated  (<a href="http://news.bbc.co.uk/1/hi/help/3223484.stm">learn more here</a>.)
-
-Some enlightened MPs are now incorporating this useful feed of their Parliamentary activity into their websites. See <a href="http://www.richardallan.org.uk/">Richard Allen's</a> website for a good example. 
-EOT
-, "2004-11-27 16:38:41"),
-
-16 => array("NEW! MPs' Expenses", <<<EOT
-At the bottom of <a href="http://www.theyworkforyou.com/mps/">each MP's page</a>, you can now see how much money your MP has claimed in allowances over the past three years.
-
-For example, it is good to note that <a href="http://www.theyworkforyou.com/mp/?pid=10508#expenses">Barbara Roche</a>, a North London MP, does not claim excessive travel expenses.
-
-Bear in mind that proper democracy does cost money, so please think twice before using these data as a stick with which to beat your MP. An "expensive" MP might be providing excellent value for money. And vice versa. 
-EOT
-, "2004-11-08 10:11:14"),
-
-11 => array("TheyWorkForYou.com Development Wiki now public", <<<EOT
-Shortly after we launched our beta test in June we decided we should also open up access to the wiki we use to help develop TheyWorkForYou.com so that anyone can also have a go, muck in, or just laugh at us.
-
-Of course, that took far longer than we planned, but having <a href="http://www.theyworkforyou.com/news/archives/2004/07/18/new_full_source_.php">published the source code</a>, we should open up the wiki too.
-
-We don't like wiki spam, so it will remain passworded, but anyone who wants a browse can tuck in at the <a href="http://www.theyworkforyou.com/wiki/moin.cgi">TheyWorkForYou.com wiki</a>
-
-username: theyworkforyou
-password: n0vemb3r (n-zero-vemb-three-r)
-
-(dissemination is fine, but please don't post the link with the login details embedded in the url)
-
-In case you're wondering what we're up to, well, we're just rousing ourselves for another tilt at completing our vision before the next election is called, so if anyone python or php skills and fancies lending a hand, do <a href="/contact/">contact us</a>.
-EOT
-, "2004-10-01 23:14:13"),
-
-8 => array("New! Full Source Code Published", <<<EOT
-<strong>Update:</strong> Code has moved location, updated the URLs and text accordingly.
-
-We're <em>really</em> keen for others to use both our code and the data feeds we make available to make the UK's Parliament more accessible.
-
-Most notably, we've published the source code for the front and back end of TheyWorkForYou.com. It's available here:
-<a href="https://secure.mysociety.org/cvstrac/dir?d=mysociety/twfy">https://secure.mysociety.org/cvstrac/dir?d=mysociety/twfy</a> for the front end website, and
-<a href="http://www.knowledgeforge.net/project/ukparse/">http://www.knowledgeforge.net/project/ukparse/</a> for the backend scraper and parser.
-
-<strong>But there's more!</strong> Below, you'll find links to various other open data feeds and resources we've produced to date:
-
-<a href="http://ukparse.kforge.net/parlparse/">http://ukparse.kforge.net/parlparse/</a> - XML files of debates / written answers back to June 2001 (House of Commons only - not Westminster Hall); MP id and constituency index files;  how to use the parser source code.
-
-<a href="http://www.publicwhip.org.uk/project/data.php">http://www.publicwhip.org.uk/project/data.php</a> - MP performance data; voting record matrices.
-
-<a href="http://theyworkforyou.com/rss/mp/10508.rdf">http://theyworkforyou.com/rss/mp/10508.rdf</a>
-    - Typical MP 'recent appearances' RSS feed, uses person id.
-EOT
-, "2004-07-18 22:58:23"),
-
-7 => array("Public Beta Now Live", <<<EOT
-Welcome to the public beta of TheyWorkForYou.com, which launched on Sunday 6th June 2004 at the <a href="http://www.notcon04.com/" title="link to NotCon04 conference website">NotCon04</a> conference.
-
-We hope you enjoy using the service during its public beta phase, which will last for  a while as we unpick all the bugs and tweak the features.  
-
-We want to know everything: what you like, what you hate, what works, what's broken, what could we do better. The lot. Don't hold back.
-
-The search engine is our main area of immediate focus - we know we've got months of tuning and tweaking of search results to come. We know enough to know that great search is hard, and that your feedback is crucial.
-
-In the meantime, please enjoy being the first people to scribble in the margins of Hansard. May you be first of many.
-
-<a href="/contact/">Send all bug reports (and feature suggestions) to us</a>.
-
-Finally, a big 'thank you' to everyone who helped test the site during the private beta phase over the past two weeks. Your feedback has been invaluable. 
-
-More than the usuals,
-
-- <i><a href="http://www.theyworkforyou.com/about/" title="link to About Us page">The TheyWorkForYou.com Volunteers</a></i> 
-EOT
-, "2004-06-06 03:02:53"),
-
-6 => array("Want to help make us complete?", <<<EOT
-Please remember that this isn't yet a complete record of our MPs' activities in the House of Commons. For that we need to add the transcripts of Select Committee proceedings and a load of other fiddly and esoteric information.  
-
-We also want to add data from before 2001. And there's always the Lords... We yearn to be a complete record.
-
-But to do this, we will need to ensure our key developers are kept fed. If you're in a sugar daddy frame of mind, we'd be only too pleased to accept donations. 
-
-Just <a href="/contact/">contact us</a>.
- 
-EOT
-, "2004-05-21 22:15:22"),
-
-5 => array("Know someone who'd like this website?", <<<EOT
-If you know someone who would appreciate being a beta tester, please <a href="/contact/">contact us</a> with their details.
-
-Many thanks. 
-EOT
-, "2004-05-21 22:55:24"),
-
-4 => array("Welcome to our private beta test", <<<EOT
-Hello all, and thank you for helping to road test TheyWorkForYou.com. We hope you enjoy our new baby; we're proud of her, even though she's still somewhat rough around the edges.
-
-We've just entered our 'closed beta' phase - you'll need a password to access the site until we move into our open beta phase sometime in June.
-
-Please <a href="/contact/">contact us</a> with your feedback, or you can just add your comments to this blog. 
-
-We want to know everything: what you like, what you hate, what works, what's broken, what could we do better. The lot. Don't hold back.
-
-The search engine is our main area of immediate focus - we know we've got months of tuning and tweaking of search results to come. We know enough to know that great search is hard, and that your feedback is crucial.
-
-In the meantime, please enjoy being the first people to scribble in the margins of Hansard. May you be first of many.
-
-- <em>The TheyWorkForYou.com volunteers</em> 
-EOT
-, "2004-05-21 22:55:24")
 );
 
 // General news functions
Index: twfy/www/docs/user/index.php
===================================================================
RCS file: /repos/mysociety/twfy/www/docs/user/index.php,v
retrieving revision 1.6
diff -u -r1.6 index.php
--- twfy/www/docs/user/index.php	7 Nov 2007 13:57:03 -0000	1.6
+++ twfy/www/docs/user/index.php	29 Nov 2007 02:59:32 -0000
@@ -328,7 +328,7 @@
 	
 	// Check postcode (which is not a compulsory field).
 	if ($details["postcode"] != "" && !validate_postcode($details["postcode"])) {
-		$errors["postcode"] = "Sorry, this isn't a valid UK postcode.";
+		$errors["postcode"] = "Sorry, this isn't a valid Australian postcode.";
 	}
 
 	// No checking of URL.
@@ -613,7 +613,7 @@
 	}
 ?>
 				<div class="row">
-				<span class="label"><label for="postcode">Your UK postcode:</label></span>
+				<span class="label"><label for="postcode">Your Australian postcode:</label></span>
 				<span class="formw"><input type="postcode" name="postcode" id="postcode" value="<?php if (isset($details["postcode"])) { echo htmlentities($details["postcode"]); } ?>" maxlength="10" size="10" class="form"> <small>Optional and not public</small></span>
 				</div>
 
Index: twfy/www/includes/dbtypes.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/dbtypes.php,v
retrieving revision 1.4
diff -u -r1.4 dbtypes.php
--- twfy/www/includes/dbtypes.php	29 Aug 2007 13:50:48 -0000	1.4
+++ twfy/www/includes/dbtypes.php	29 Nov 2007 02:59:32 -0000
@@ -13,7 +13,7 @@
 $hansardmajors = array(
 	1 => array(
 		'type'=>'debate',
-		'title'=>"Commons debates",
+		'title'=>"House debates",
 		'singular'=>'debate',
 		'plural'=>'debates',
 		'page'=>'debate',
Index: twfy/www/includes/easyparliament/config.php.incvs
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/config.php.incvs,v
retrieving revision 1.7
diff -u -r1.7 config.php.incvs
--- twfy/www/includes/easyparliament/config.php.incvs	6 Nov 2007 13:30:11 -0000	1.7
+++ twfy/www/includes/easyparliament/config.php.incvs	29 Nov 2007 02:59:32 -0000
@@ -56,6 +56,8 @@
 // Web path to the directory where the site's images are kept.
 define ("IMAGEPATH", WEBPATH . "images/");
 
+// Filesystem path to where the images are kept
+define ("FILEIMAGEPATH", BASEDIR . "images/");
 
 // This will be included in data.php. It's an array of page/section metadata.
 define ("METADATAPATH", BASEDIR . "/../includes/easyparliament/metadata.php");
Index: twfy/www/includes/easyparliament/metadata.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/metadata.php,v
retrieving revision 1.10
diff -u -r1.10 metadata.php
--- twfy/www/includes/easyparliament/metadata.php	19 Oct 2007 09:23:42 -0000	1.10
+++ twfy/www/includes/easyparliament/metadata.php	29 Nov 2007 02:59:32 -0000
@@ -284,11 +284,11 @@
 	),			
 	'debatesfront' => array (
 		'menu'			=> array (
-			'text'			=> 'Commons Debates',
-			'title'			=> "House of Commons debates"
+			'text'			=> 'House Debates',
+			'title'			=> "House debates"
 		),
 		'parent'		=> 'hansard',
-		'title'			=> 'House of Commons debates',
+		'title'			=> 'House debates',
 		'track'			=> true,
 		'rss'			=> 'debates/debates.rss',
 		'url'			=> 'debates/'
@@ -337,15 +337,15 @@
 	'hansard' => array (
 		'menu'			=> array (
 			'text'			=> 'Hansard',
-			'title'			=> "House of Commons debates and Written Answers"
+			'title'			=> "House debates"
 		),
-		'title'			=> 'House of Commons, House of Lords, and Northern Ireland Assembly',
+		'title'			=> 'House of Representatives',
 		'track'			=> true,
 		'url'			=> 'hansard/'
 	),	
 	'hansard_date' => array (
 		'parent'		=> 'hansard',
-		'title'			=> 'House of Commons, House of Lords, and Northern Ireland Assembly',
+		'title'			=> 'House of Representatives',
 		'track'			=> true,
 		'url'			=> 'hansard/'
 	),	
@@ -482,10 +482,10 @@
 
 	'mps' => array (
 		 'menu'			=> array (
-			'text'			=> 'All MPs',
-			'title'			=> "List of all MPs"
+			'text'			=> 'All Representatives',
+			'title'			=> "List of all Members of the House of Representatives"
 		),
-		'title'			=> 'All MPs',
+		'title'			=> 'All Members of the House of Representatives',
 		'track'			=> true,
 		'url'			=> 'mps/'
 	),
@@ -854,8 +854,8 @@
 
 	'yourmp' => array (
 		'menu'			=> array (
-			'text'			=> 'Your MP',
-			'title'			=> "Find out about your Member of Parliament"
+			'text'			=> 'Your Representative',
+			'title'			=> "Find out about your Member of the House of Representatives"
 		),
 		'sidebar'		=> 'yourmp',
 		'title'			=> 'Your MP',
Index: twfy/www/includes/easyparliament/page.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/page.php,v
retrieving revision 1.76
diff -u -r1.76 page.php
--- twfy/www/includes/easyparliament/page.php	15 Nov 2007 17:21:26 -0000	1.76
+++ twfy/www/includes/easyparliament/page.php	29 Nov 2007 02:59:33 -0000
@@ -239,7 +239,7 @@
 type="text/javascript">
 </script>
 <script type="text/javascript">
-_uacct = "UA-660910-1";
+_uacct = "UA-3107958-1";
 if (typeof urchinTracker == 'function') urchinTracker();
 </script>
 
@@ -336,11 +336,11 @@
 		// we're within that section.
 		$items = array (
 			'home' 		=> array ('sitenews', 'comments_recent', 'api_front'),
-			'hansard' 	=> array ('debatesfront', 'wransfront', 'whallfront', 'wmsfront', 'lordsdebatesfront', 'nidebatesfront'),
+			'hansard' 	=> array ('debatesfront'),
 			'yourmp'	=> array (),
 			'mps'           => array (),
-			'peers'		=> array (),
-			'mlas'          => array (),
+#			'peers'		=> array (),
+#			'mlas'          => array (),
 #			'help_us_out'	=> array (), 
 /*			'help_us_out'	=> array ('glossary_addterm'),  */
 			'help'		=> array ()
@@ -933,7 +933,7 @@
 		?>
 				<br>
 <?php
-		$this->block_start(array('id'=>'mp', 'title'=>'Find out about your MP'));
+		$this->block_start(array('id'=>'mp', 'title'=>'Find out about your Representative'));
 		?>
 						<form action="<?php echo $MPURL->generate(); ?>" method="get">
 <?php
@@ -948,9 +948,9 @@
 <?php
 		}
 		?>
-						<p><strong>Enter your UK postcode: </strong>
+						<p><strong>Enter your Australian postcode: </strong>
 
-						<input type="text" name="pc" value="<?php echo htmlentities(get_http_var('pc')); ?>" maxlength="10" size="10"> <input type="submit" value="GO" class="submit"> <small>(e.g. BS3 1QP)</small>
+						<input type="text" name="pc" value="<?php echo htmlentities(get_http_var('pc')); ?>" maxlength="10" size="10"> <input type="submit" value="GO" class="submit"> <small>(e.g. 2340)</small>
 						</p>
 						<input type="hidden" name="ch" value="t">
 						</form>
@@ -1031,16 +1031,16 @@
 		}
 		print '<p class="printonly">This data was produced by TheyWorkForYou from a variety of sources.</p>';
 		$this->block_start(array('id'=>'mp', 'title'=>$title));
-		if (is_file(BASEDIR . IMAGEPATH . 'mpsL/' . $member['person_id'] . '.jpg')) {
+		if (is_file(FILEIMAGEPATH . 'mpsL/' . $member['person_id'] . '.jpg')) {
 		?>
 						<img src="<?php echo IMAGEPATH;?>mpsL/<?php echo $member['person_id']; ?>.jpg" alt="Photo of <?php echo $member['full_name']; ?>" class="portrait">
-		<? } elseif (is_file(BASEDIR . IMAGEPATH . 'mpsL/' . $member['person_id'] . '.jpeg')) {
+		<? } elseif (is_file(FILEIMAGEPATH . 'mpsL/' . $member['person_id'] . '.jpeg')) {
 		?>
 						<img src="<?php echo IMAGEPATH;?>mpsL/<?php echo $member['person_id']; ?>.jpeg" alt="Photo of <?php echo $member['full_name']; ?>" class="portrait">
-		<? } elseif (is_file(BASEDIR . IMAGEPATH . 'mps/' . $member['person_id'] . '.jpg')) {
+		<? } elseif (is_file(FILEIMAGEPATH . 'mps/' . $member['person_id'] . '.jpg')) {
 		?>
 						<img src="<?php echo IMAGEPATH;?>mps/<?php echo $member['person_id']; ?>.jpg" alt="Photo of <?php echo $member['full_name']; ?>" height="118" class="portrait">
-		<? } elseif (is_file(BASEDIR . IMAGEPATH . 'mps/' . $member['person_id'] . '.jpeg')) {
+		<? } elseif (is_file(FILEIMAGEPATH . 'mps/' . $member['person_id'] . '.jpeg')) {
 		?>
 						<img src="<?php echo IMAGEPATH;?>mps/<?php echo $member['person_id']; ?>.jpeg" alt="Photo of <?php echo $member['full_name']; ?>" height="118" class="portrait">
 		<? } ?>
@@ -1201,8 +1201,6 @@
 <?php
 		} elseif ($member['current_member'][1]) {
 			?>
-						<li><a href="http://www.writetothem.com/"><strong>Send a message to your MP</strong></a> <small>(via WriteToThem.com)</small></li>
-						<li><a href="http://www.hearfromyourmp.com/"><strong>Sign up to <em>HearFromYourMP</em></strong></a> to get messages from your MP</li>
 <?php
 		} elseif ($member['current_member'][3]) {
 			?>
@@ -1225,7 +1223,6 @@
 						
 						<ul class="jumpers">
 <? if ((in_array(1, $member['houses']) && $member['party']!='Sinn Fein') || in_array(2, $member['houses'])) { ?>
-						<li><a href="#votingrecord">Voting record</a></li>
 <?		if (in_array(1, $member['houses'])) { ?>
 						<li><a href="#topics">Committees and topics of interest</a></li>
 <?		} ?>
@@ -1252,7 +1249,7 @@
 		$this->block_end();
 
 # Big don't-print for SF MPs
-if ((in_array(1, $member['houses']) && $member['party']!='Sinn Fein') || in_array(2, $member['houses'])) {
+if (0) {
 
 		// Voting Record.
 		?> <a name="votingrecord"></a> <?php
@@ -1515,7 +1512,7 @@
 			$minister = true;
 		if (isset($extra_info['Lwrans_answered_inlastyear']) && $extra_info['Lwrans_answered_inlastyear'] > 0 && $extra_info['Lwrans_asked_inlastyear'] == 0)
 			$Lminister = true;
-		$displayed_stuff |= display_stats_line('wrans_asked_inlastyear', 'Has received answers to <a href="' . $MOREURL->generate() . '">', 'written question', '</a> ' . $since_text, '', $extra_info, $minister, $Lminister);
+#		$displayed_stuff |= display_stats_line('wrans_asked_inlastyear', 'Has received answers to <a href="' . $MOREURL->generate() . '">', 'written question', '</a> ' . $since_text, '', $extra_info, $minister, $Lminister);
 		}
 
 		if (isset($extra_info['select_committees'])) {
@@ -1527,17 +1524,17 @@
 			print '.</li>';
 		}
 
-		$wtt_displayed = display_writetothem_numbers(2006, $extra_info);
-		$displayed_stuff |= $wtt_displayed;
-		if (!$wtt_displayed)
-			$displayed_stuff |= display_writetothem_numbers(2005, $extra_info);
+#		$wtt_displayed = display_writetothem_numbers(2006, $extra_info);
+#		$displayed_stuff |= $wtt_displayed;
+#		if (!$wtt_displayed)
+# 			$displayed_stuff |= display_writetothem_numbers(2005, $extra_info);
 
 		$after_stuff = ' <small>(From Public Whip)</small>';
 		if ($member['party'] == 'Scottish National Party') {
 			$after_stuff .= '<br><em>Note SNP MPs do not vote on legislation not affecting Scotland.</em>';
 		}
 		if ($member['party'] != 'Sinn Fein') {
-			$displayed_stuff |= display_stats_line('public_whip_division_attendance', 'Has voted in <a href="http://www.publicwhip.org.uk/mp.php?id=uk.org.publicwhip/member/' . $member['member_id'] . '&amp;showall=yes#divisions" title="See more details at Public Whip">', 'of vote', '</a> in parliament', $after_stuff, $extra_info);
+#			$displayed_stuff |= display_stats_line('public_whip_division_attendance', 'Has voted in <a href="http://www.publicwhip.org.uk/mp.php?id=uk.org.publicwhip/member/' . $member['member_id'] . '&amp;showall=yes#divisions" title="See more details at Public Whip">', 'of vote', '</a> in parliament', $after_stuff, $extra_info);
 
 			$displayed_stuff |= display_stats_line('comments_on_speeches', 'People have made <a href="' . WEBPATH . 'comments/recent/?pid='.$member['person_id'].'">', 'comment', "</a> on this MP's speeches", '', $extra_info);
 			$displayed_stuff |= display_stats_line('reading_age', 'This MP\'s speeches are understandable to an average ', '', ' year old, going by the <a href="http://en.wikipedia.org/wiki/Flesch-Kincaid_Readability_Test">Flesch-Kincaid Grade Level</a> score', '', $extra_info);
Index: twfy/www/includes/easyparliament/people.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/people.php,v
retrieving revision 1.9
diff -u -r1.9 people.php
--- twfy/www/includes/easyparliament/people.php	27 Jul 2007 14:30:10 -0000	1.9
+++ twfy/www/includes/easyparliament/people.php	29 Nov 2007 02:59:33 -0000
@@ -17,7 +17,7 @@
 	function display ($view, $args=array(), $format='html') {
 		global $PAGE;
 	
-		$validviews = array('mps', 'peers', 'mlas');
+		$validviews = array('mps', 'mlas');
 		
 		if (in_array($view, $validviews)) {
 		
Index: twfy/www/includes/easyparliament/user.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/user.php,v
retrieving revision 1.7
diff -u -r1.7 user.php
--- twfy/www/includes/easyparliament/user.php	15 Nov 2007 18:49:08 -0000	1.7
+++ twfy/www/includes/easyparliament/user.php	29 Nov 2007 02:59:34 -0000
@@ -295,7 +295,7 @@
 
 		$urltoken = $this->user_id . '-' . $this->registrationtoken;
 
-		$confirmurl = 'http://' . DOMAIN . '/U/' . $urltoken;
+		$confirmurl = 'http://' . DOMAIN . WEBPATH . 'U/' . $urltoken;
 
 		// Arrays we need to send a templated email.
 		$data = array (
Index: twfy/www/includes/easyparliament/sidebars/donate.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/sidebars/donate.php,v
retrieving revision 1.3
diff -u -r1.3 donate.php
--- twfy/www/includes/easyparliament/sidebars/donate.php	10 Aug 2007 11:09:19 -0000	1.3
+++ twfy/www/includes/easyparliament/sidebars/donate.php	29 Nov 2007 02:59:34 -0000
@@ -4,10 +4,9 @@
 $this->block_start(array('id'=>'help', 'title'=>"Did you know this site is run by a charity?"));
 ?>
 
-<p><a href="https://secure.mysociety.org/donate/"><img align="right" src="/images/donate_red_flatL.gif" width="100" height="35" border="0" hspace="4" vspace="5" alt="Donate"></a>
-We've just added the Northern Ireland Assembly, email alerts
-have been sped up dramatically &mdash; your donation would enable
-us to continue to add new features, such as committees.</p>
+<p><a href="https://secure.mysociety.org/donate/"><img align="right" src=<?php echo WEBPATH . "images/donate_red_flatL.gif"?> width="100" height="35" border="0" hspace="4" vspace="5" alt="Donate"></a>
+Your donation would enable
+us to continue to add new features.</p>
 
 <?php
 $this->block_end();
Index: twfy/www/includes/easyparliament/sidebars/hocdebates.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/sidebars/hocdebates.php,v
retrieving revision 1.1
diff -u -r1.1 hocdebates.php
--- twfy/www/includes/easyparliament/sidebars/hocdebates.php	27 Apr 2006 14:20:40 -0000	1.1
+++ twfy/www/includes/easyparliament/sidebars/hocdebates.php	29 Nov 2007 02:59:34 -0000
@@ -6,7 +6,7 @@
 $this->block_start(array('id'=>'help', 'title'=>"What are Debates?"));
 ?>
 
-<p><strong>Debates</strong> in the House of Commons are an opportunity for MPs from all parties to <strong>scrutinise</strong> government legislation and <strong>raise important local, national or topical issues</strong>.</p>
+<p><strong>Debates</strong> in the House of Representatives are an opportunity for members from all parties to <strong>scrutinise</strong> government legislation and <strong>raise important local, national or topical issues</strong>.</p>
 <p>And sometimes to shout at each other.</p>
 <p>
 <!--For more about debates, click <a href="<?php echo $helpurl; ?>#debates">here</a>.-->
Index: twfy/www/includes/easyparliament/sidebars/hocdebates_short.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/sidebars/hocdebates_short.php,v
retrieving revision 1.1
diff -u -r1.1 hocdebates_short.php
--- twfy/www/includes/easyparliament/sidebars/hocdebates_short.php	27 Apr 2006 14:20:40 -0000	1.1
+++ twfy/www/includes/easyparliament/sidebars/hocdebates_short.php	29 Nov 2007 02:59:34 -0000
@@ -3,8 +3,8 @@
 
 $this->block_start(array(
 	'id'=>'help', 
-	'title'=>"What are Commons debates?", 
-	'url'=>'/debates/#help',
+	'title'=>"What are House debates?", 
+	'url'=> WEBPATH . 'debates/#help',
 	'body'=>false
 ));
 $this->block_end();
Index: twfy/www/includes/easyparliament/sidebars/mps.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/sidebars/mps.php,v
retrieving revision 1.2
diff -u -r1.2 mps.php
--- twfy/www/includes/easyparliament/sidebars/mps.php	23 Jun 2006 23:18:20 -0000	1.2
+++ twfy/www/includes/easyparliament/sidebars/mps.php	29 Nov 2007 02:59:34 -0000
@@ -9,7 +9,7 @@
 	'body'=>''));
 ?>
 <p>
-	Vacher Dods charge <a href="http://www.dodonline.co.uk/engine.asp?showPage=article&id=2564"> 95 pounds </a> for this list of MPs names. We think democracy should be free. Click <?php echo '<a href="'.$csvurl.'">here</a>'; ?> to download a CSV (Comma Separated Values) file that you can load into Excel.
+	Click <?php echo '<a href="'.$csvurl.'">here</a>'; ?> to download a CSV (Comma Separated Values) file that you can load into Excel.
 </p>
 <?php
 $this->block_end();
Index: twfy/www/includes/easyparliament/sidebars/whatisthissite.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/sidebars/whatisthissite.php,v
retrieving revision 1.4
diff -u -r1.4 whatisthissite.php
--- twfy/www/includes/easyparliament/sidebars/whatisthissite.php	10 Aug 2007 11:09:19 -0000	1.4
+++ twfy/www/includes/easyparliament/sidebars/whatisthissite.php	29 Nov 2007 02:59:34 -0000
@@ -10,7 +10,7 @@
 $helpurl = $URL->generate();
 ?>
 
-<p><a href="https://secure.mysociety.org/donate/"><img src="/images/donate_red_flatL.gif" width="100" height="35" border="0" align="right" hspace="4" vspace="5" alt="Donate"></a>
+<p><a href="https://secure.mysociety.org/donate/"><img src=<?php echo WEBPATH . "images/donate_red_flatL.gif" ?> width="100" height="35" border="0" align="right" hspace="4" vspace="5" alt="Donate"></a>
 <a href="<?php echo $abouturl; ?>" title="link to About Us page">TheyWorkForYou.com</a>
 is a non-partisan website run by a charity which aims to
 make it easy for people to keep tabs on their elected and
Index: twfy/www/includes/easyparliament/staticpages/linktous.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/staticpages/linktous.php,v
retrieving revision 1.2
diff -u -r1.2 linktous.php
--- twfy/www/includes/easyparliament/staticpages/linktous.php	10 Aug 2007 11:09:19 -0000	1.2
+++ twfy/www/includes/easyparliament/staticpages/linktous.php	29 Nov 2007 02:59:34 -0000
@@ -6,9 +6,9 @@
 <!-- TheyWorkForYou box, begin -->
 <div style="position: relative; width: 17em; color: #000000; background-color: #E8FDCB; font-family: Arial, Geneva, Sans-serif; margin-bottom: 1em; border: 1px solid #4d6c25; padding: 0 10px 2em 10px;">
 	<h5 style="font-family: Arial, Geneva, Sans-serif; font-size: 1.4em; position: absolute; margin: 0; bottom: 2px; right: 10px;"  title="TheyWorkForYou.com"><a style="color: #880101; text-decoration: none; font-weight: normal;" href="http://www.TheyWorkForYou.com"><em style="font-weight: bold; font-style: normal;">TheyWorkFor</em><strong style="color: #cc0000; font-weight: bolder;">You</strong>.com</a></h5>
-	<form style="margin: 0; padding: 5px 0 0 0;" title="Find out about your MP" action="http://www.theyworkforyou.com/mp/" method="get">
+	<form style="margin: 0; padding: 5px 0 0 0;" title="Find out about your Representative" action="http://www.theyworkforyou.com/mp/" method="get">
 		<label style="display: block; font-size: small; font-weight: bold; margin: 0 0 2px 0;" for="pc">Find out more about your <abbr class="display: inline; speak: spell-out;" title="Member of Parliament">MP</abbr></label>
-		<input id="pc" name="pc" style="width: 12em; border: solid 1px #4D6C25;" title="Enter your UK postcode here" tabindex="1" type="text" size="8" maxlength="20" value="Your Postcode"> 
+		<input id="pc" name="pc" style="width: 12em; border: solid 1px #4D6C25;" title="Enter your Australian postcode here" tabindex="1" type="text" size="8" maxlength="20" value="Your Postcode"> 
 		<input id="Submit1" name="Submit1" style="border: solid 0px #4d6c25; background-color: #4D6C25; color: #ffffff; font-weight: bold; text-transform: uppercase;" title="Submit search" tabindex="2" type="submit" value="Go">
 	</form>
 	<form style="margin: 0; padding: 5px 0 0 0;" title="Search everything said in Parliament since 2001" action="http://www.theyworkforyou.com/search/" method="get">
Index: twfy/www/includes/easyparliament/templates/api/hansard_gid.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/templates/api/hansard_gid.php,v
retrieving revision 1.1
diff -u -r1.1 hansard_gid.php
--- twfy/www/includes/easyparliament/templates/api/hansard_gid.php	25 Aug 2006 19:38:31 -0000	1.1
+++ twfy/www/includes/easyparliament/templates/api/hansard_gid.php	29 Nov 2007 02:59:34 -0000
@@ -18,9 +18,9 @@
 		if ($row['htype'] == '12') {
 			if (isset($row['speaker']) && count($row['speaker']) > 0) {
 				$speaker = $row['speaker'];
-				if (is_file(BASEDIR . IMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpeg')) {
+				if (is_file(FILEIMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpeg')) {
 					$row['speaker']['image'] = IMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpeg';
-				} elseif (is_file(BASEDIR . IMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpg')) {
+				} elseif (is_file(FILEIMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpg')) {
 					$row['speaker']['image'] = IMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpg';
 				}
 				$desc = '';
Index: twfy/www/includes/easyparliament/templates/emails/alert_confirmation.txt
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/templates/emails/alert_confirmation.txt,v
retrieving revision 1.2
diff -u -r1.2 alert_confirmation.txt
--- twfy/www/includes/easyparliament/templates/emails/alert_confirmation.txt	11 Aug 2006 20:55:57 -0000	1.2
+++ twfy/www/includes/easyparliament/templates/emails/alert_confirmation.txt	29 Nov 2007 02:59:34 -0000
@@ -1,5 +1,5 @@
-Subject: Please confirm your TheyWorkForYou.com alert
-You have asked to subscribe to an alert at TheyWorkForYou.com
+Subject: Please confirm your OpenAustralia.org alert
+You have asked to subscribe to an alert at OpenAustralia.org
 for the following criteria:
 
 {CRITERIA}
@@ -14,11 +14,11 @@
 
 If you have not tried to register for this alert then someone else
 entered your address. If you think this has been done maliciously
-then let us know at volunteers@theyworkforyou.com
+then let us know at matthew@openaustralia.org
 
 And don't forget to let your friends and family know about
-TheyWorkForYou.com 
+OpenAustralia.org
 
 Best wishes,
 
-- The TheyWorkForYou.com Volunteers
+- The OpenAustralia.org Volunteers
Index: twfy/www/includes/easyparliament/templates/emails/join_confirmation.txt
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/templates/emails/join_confirmation.txt,v
retrieving revision 1.1
diff -u -r1.1 join_confirmation.txt
--- twfy/www/includes/easyparliament/templates/emails/join_confirmation.txt	27 Apr 2006 14:20:41 -0000	1.1
+++ twfy/www/includes/easyparliament/templates/emails/join_confirmation.txt	29 Nov 2007 02:59:34 -0000
@@ -1,7 +1,7 @@
-Subject: Please confirm your TheyWorkForYou.com account
+Subject: Please confirm your OpenAustralia.org account
 Hi {FIRSTNAME},
 
-Welcome to TheyWorkForYou.com, the website which lets you scribble all over the margins of Hansard.
+Welcome to OpenAustralia.org, the website which lets you scribble all over the margins of Hansard.
 
 Before you can log in and add comments, we need you to confirm your registration by clicking on the link below:
 
@@ -9,10 +9,10 @@
 
 If clicking the link above doesn't work, you may have to copy it and then paste it into the 'Location' or 'Address' box of your web browser, and then press 'enter' or 'return' on your keyboard.
 
-Please feel free to email us at volunteers@theyworkforyou.com with any comments, problems or suggestions. 
+Please feel free to email us at matthew@openaustralia.org with any comments, problems or suggestions. 
 
-And don't forget to let your friends and family know about TheyWorkForYou.com 
+And don't forget to let your friends and family know about OpenAustralia.org 
 
 Best wishes,
 
-- The TheyWorkForYou.com Volunteers
+- The OpenAustralia.org Volunteers
Index: twfy/www/includes/easyparliament/templates/html/hansard_gid.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/templates/html/hansard_gid.php,v
retrieving revision 1.13
diff -u -r1.13 hansard_gid.php
--- twfy/www/includes/easyparliament/templates/html/hansard_gid.php	29 Aug 2007 13:50:49 -0000	1.13
+++ twfy/www/includes/easyparliament/templates/html/hansard_gid.php	29 Nov 2007 02:59:34 -0000
@@ -202,12 +202,12 @@
 				$speakername = ucfirst(member_full_name($speaker['house'], $speaker['title'], $speaker['first_name'], $speaker['last_name'], $speaker['constituency']));
 				?>
 				<p class="speaker"><a href="<?php echo $speaker['url']; ?>" title="See more information about <?php echo $speakername; ?>">
-<? if (is_file(BASEDIR . IMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpeg')) { ?>
+<? if (is_file(FILEIMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpeg')) { ?>
                 <img src="<?php echo IMAGEPATH . 'mps/' . $speaker['person_id']; ?>.jpeg" class="portrait" alt="Photo of <?php echo $speakername; ?>" 
 				<?php if (get_http_var('partycolours')) { ?>
                 style="border: 3px solid <?php echo party_to_colour($speaker['party']) ?>;"
                 <?php } ?>
-		><?php } elseif (is_file(BASEDIR . IMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpg')) { ?>
+		><?php } elseif (is_file(FILEIMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpg')) { ?>
 		<img src="<?php echo IMAGEPATH . 'mps/' . $speaker['person_id']; ?>.jpg" class="portrait" alt="Photo of <?php echo $speakername; ?>"><? } ?>
                 <strong><?php echo $speakername; ?></strong></a> <small><?php 
 				$desc = '';
